tools/test_yaml_valid

```diff
@@ -1,7 +1,16 @@
 #!/usr/bin/env perl
 use Test::More;
-use Test::YAML::Valid qw(-Tiny);
+use YAML::PP;
 
 pass("No yaml files modified") unless @ARGV;
-yaml_file_ok($_, $_) for @ARGV;
+for my $file (@ARGV) {
+    eval { YAML::PP::LoadFile($file) };
+    if (my $err = $@) {
+        fail("$file valid");
```

Comment:
do you mean invalid here?

Reply:
Good point, should be invalid in this case.

Reply:
Fixed now, thanks for raising.

Reply:
Well, the test is "$file valid", and in this case it failed the test that it is valid.

"Failed test $file invalid" would sound wrong.

But if you prefer it...